src/components/FetchRouteList.tsx

```diff
@@ -0,0 +1,70 @@
+import { Badge, Text } from "@fluentui/react-components";
+import { Card } from "@fluentui/react-components/unstable";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { RoutesXml } from "../data/etaXml";
+import RawDisplay from "./RawDisplay";
+import { FetchXMLWithCancelToken } from "./fetchUtils";
+
+const parseRouteTitle = (input: string) => {
+  const routeTitleRegex = /\d+-/;
+  if (routeTitleRegex.test(input)) {
+    return input.replace(routeTitleRegex, "");
+  } else {
+    return input;
+  }
+};
+
+export function RoutesInfo() {
+  const controller = new AbortController();
+  const [data, setData] = useState<RoutesXml>();
+  const [routesDb, setRoutesDb] = useState<{ tag: number; title: string }[]>(
```

Comment:
should we type it?

Reply:
right now it looks simple and not repeated elsewhere; keeping it in file helps with clarity

will make an interface if it is reused